### ISSUE TITLE
[Feat.] Add padding before batch postprocess in agentloop

### ISSUE THREAD

**Issue by PopSoda2002** [volcengine/verl#2677]
Previously, we will pad the prompts, responses, response_mask in the ``postprocess`` for the batches of ``AgentLoopOutput.``
```

    def _postprocess(self, inputs: list[AgentLoopOutput]) -> DataProto:

        # prompts
        self.tokenizer.padding_side = "left"
        outputs = self.tokenizer.pad(
            [{"input_ids": input.prompt_ids} for input in inputs],
            padding="max_length",
            max_length=self.config.actor_rollout_ref.rollout.prompt_length,
            return_tensors="pt",
            return_attention_mask=True,
        )
        prompt_ids, prompt_attention_mask = outputs["input_ids"], outputs["attention_mask"]

        # responses
        self.tokenizer.padding_side = "right"
        outputs = self.tokenizer.pad(
            [{"input_ids": input.response_ids} for input in inputs],
            padding="max_length",
            max_length=self.config.actor_rollout_ref.rollout.response_length,
            return_tensors="pt",
            return_attention_mask=True,
        )
        response_ids, response_attention_mask = outputs["input_ids"], outputs["attention_mask"]

        # response_mask
        outputs = self.tokenizer.pad(
            [{"input_ids": input.response_mask} for input in inputs],
            padding="max_length",
            max_length=self.config.actor_rollout_ref.rollout.response_length,
            return_tensors="pt",
            return_attention_mask=False,
        )
```
Sometimes, the padding process can be time-consuming. So we plan to directlt pad after getting the generation output for the single prompt, and concat the results together after that to save time:

![Image](https://github.com/user-attachments/assets/d41f7f9f-8b0e-464c-b94a-deade43b866e)

And we will profile for this change!

**Comment by PopSoda2002** [comment]
I will take this

**Comment by PopSoda2002** [comment]
Solved in https://github.com/volcengine/verl/pull/2773
